File: share-code/bai-tap-lap-trinh/python/condition/46.py
total_lines = 8
time_doing = 4

SALARY_ONE_HOUR         = 20
PRICE_REVIEW_LEVEL_ONE  = 1
PRICE_REVIEW_LEVEL_TWO  = 2
PRICE_REVIEW_LEVEL_THREE= 5

review_fee = PRICE_REVIEW_LEVEL_THREE

if total_lines < 10:
    review_fee = PRICE_REVIEW_LEVEL_ONE
elif 10 <= total_lines <= 20:
    review_fee = PRICE_REVIEW_LEVEL_TWO

total_income = time_doing * SALARY_ONE_HOUR - review_fee

print(f"Salary: {total_income}$")

File: share-code/bai-tap-lap-trinh/cpp/46.cpp
#include <iostream>
using namespace std;
int main()
{
    int totalLine                        = 50;
    int timeDoing                        = 4;
    const int SALARY_ONE_HOUR            = 20;
    const int PRICE_REVIEW_LEVEL_ONE     = 1;
    const int PRICE_REVIEW_LEVEL_TWO     = 2;
    const int PRICE_REVIEW_LEVEL_THREE   = 5;
    int priceReview                      = PRICE_REVIEW_LEVEL_THREE;

    if (totalLine < 10)
    {
        priceReview = PRICE_REVIEW_LEVEL_ONE;
    }else if (totalLine >= 10 && totalLine < 20) // [10 , 20)
    {
        priceReview = PRICE_REVIEW_LEVEL_TWO;
    }

    // lương = thgian làm việc x lương trong 1 giờ - chi phí review;
    int salary = timeDoing * SALARY_ONE_HOUR - priceReview;
    cout << "Salary: " << salary;
}